### ISSUE TITLE
@nuxtjs/partytown  is not working with nuxt 3

### ISSUE THREAD

**Issue by anish2690** [nuxt-community/partytown-module#58]
```ts
import { defineNuxtConfig } from 'nuxt3'

export default defineNuxtConfig({
  modules: ['@nuxtjs/partytown'],
  partytown: {
    forward: ['$crisp', '$crisp.push'],
    debug: true
  },
  meta: {
    script: [
      // Insert your CRISP Script here e.g.:
      { children: 'window.$crisp = []; window.CRISP_WEBSITE_ID = "0000"' },
      { src: 'https://client.crisp.chat/l.js', async: true, type: 'text/partytown' } 
    ]
  }
})
```
Worker not getting initialised 

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/22445409/163234681-60293611-42f1-4604-b829-7354d8115a01.png">

**Comment by danielroe** [comment]
This is the issue: https://github.com/nuxt/framework/pull/4304 - the `nitro:generate` hook was removed.

**Comment by TrungRueta** [comment]
maybe use `nitro:start` will be solution ?

**Comment by danielroe** [comment]
Thanks for the issue - we're now using new nitro publicAssets to directly add + serve partytown directory: https://github.com/nuxt-community/partytown-module/commit/bf4cfca4df10a009abf2c16bfc7edf7be0a4efeb.

Issue should be resolved in 0.1.1.
